descartes_utilities/include/descartes_utilities/ros_conversions.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2016, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ *  ros_conversions.h
+ *
+ *  Created on: Feb 28, 2016
+ *  Author: Jonathan Meyer
+ */
+
+#ifndef DESCARTES_ROS_CONVERSIONS_H
+#define DESCARTES_ROS_CONVERSIONS_H
+
+#include <trajectory_msgs/JointTrajectory.h>
+#include <descartes_core/trajectory_pt.h>
+
+namespace descartes_utilities
+{
+  /**
+   * @brief Converts a sequence of Descartes joint trajectory points to ROS trajectory points.
```

Comment:
We ran into this issues with this, when trying to execute trajectories - converted using any of the 'boilerplate' versions of the descartes-to-moveit code - on robots other than the UR.

The code in the [roscon_2015](https://github.com/ros-industrial-consortium/roscon_2015) repository generates trajectories in the same way, and it seems to only result in smooth motion due to the dynamics of the UR and the relatively short traj segments. If the timing of the individual segments is changed, we quickly experienced the UR performing a sort of *stop-motion*. I expect the same for drivers like `motoman_driver`: which depend heavily on the values in the `velocity` and `acceleration` fields to be able to resample the incoming segments.

We sometimes use the `IterativeParabolicTimeParameterization` from MoveIt to fixup trajectories, but that's all in joint space.